ross/fluid_flow/fluid_flow_coefficients.py

```diff
@@ -182,14 +93,17 @@ def calculate_oil_film_force(fluid_flow_object, force_type=None):
     return radial_force, tangential_force, force_x, force_y
 
 
-def calculate_stiffness_matrix(fluid_flow_object, oil_film_force='numerical'):
+def calculate_stiffness_matrix(fluid_flow_object, force_type=None, oil_film_force='numerical'):
     """This function calculates the bearing stiffness matrix numerically.
     Parameters
     ----------
     fluid_flow_object: A FluidFlow object.
     oil_film_force: str
-        If set, calculates the oil film force analytically considering the chosen type: 'short' or 'long'.
+        If set, calculates the oil film force analytically analytically considering the chosen type: 'short'.
         If set to 'numerical', calculates the oil film force numerically.
+    force_type: str
+        If set, calculates the stiffness matrix analytically considering the chosen type: 'short'.
+        If set to 'numerical', calculates the stiffness matrix numerically.
```

Comment:
I didn't understand well these docstrings.
The options to `force_type` attribute are "short" and "numerical"? The same comment goes to `oir_film_force`.
Maybe we could set `force_type` with a default argument

Also, you could write the input options, like: 
```
"""
Options are: "short" and "long".
Default is "short"
"""
``` 

At line 102, analytically is repeated.

Reply:
The approximations presented by Friswell for the stiffness and damping matrices are only for the case of the short bearing. So the options for force_type attribute are "short" and "numerical".

For oil_film_force we really have both cases. It was a mistake. I will correct that.